packaging/rpm/flightctl.spec

```diff
@@ -414,7 +414,11 @@ echo "Flightctl Observability Stack uninstalled."
     SOURCE_GIT_TREE_STATE=clean \
     SOURCE_GIT_COMMIT=$(echo %{version} | awk -F'[-~]g' '{print $2}') \
     SOURCE_GIT_TAG_NO_V=%{version} \
-    make build-cli build-agent
+    %if 0%{?rhel} == 9
```

Comment:
what about rhel 10 should we do `>=`

Reply:
We probably should, but building for RHEL 10 will fail elsewhere first I’m afraid.